auth0/resource_auth0_connection.go

```diff
@@ -752,7 +752,7 @@ func connectionSchemaUpgradeV1(
 
 		m["validation"] = []map[string][]interface{}{
 			{
-				"username": []interface{}{validationOption.(interface{})},
+				"username": []interface{}{validationOption},
```

Comment:
This was always already an `interface{}` no need to typecast it twice due to https://github.com/auth0/terraform-provider-auth0/pull/130/files#diff-83d4cd14f38b8b51fbd061412cc8852def1cdb24f87fffea64d93c876c146987L748.